.github/workflows/tests-ubuntu.yml

```
@@ -26,19 +26,19 @@ jobs:
         - python-version: pypy3
           env:
             TOXENV: pypy3
-            PYPY_VERSION: 3.6-v7.3.3
+            PYPY_VERSION: 3.7-v7.3.9
```

I believe this is meant to be the latest PyPy release, [3.6 just happened to be the latest Python version available at the time](https://doc.pypy.org/en/latest/release-v7.3.1.html) when [this was first defined](https://github.com/scrapy/scrapy/commit/80db569aeae3e05480ed228c95332c17f67e81e4), and since then [we only updated it once to address a specific issue](https://github.com/scrapy/scrapy/pull/5190/commits/ba40bccc2d779d332cc74f43759fe57f71d3376e), and being focused on addressing the issue we forgot to update to Python 3.7 back then.

```suggestion
            PYPY_VERSION: 3.9-v7.3.9
```